webnn/resources/test_data/concat.json

```diff
@@ -0,0 +1,1713 @@
+{
+  "tests": [
+    // concat 1D tensors
+    {
+      "name": "concat two 1D of same shape along axis 0",
+      "inputs": [
```

Comment:
Here `inputs` is an array for first parameter of `concat()`, to create input operands easily, I use operand name likes input1, input2, input3, ... .